sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/AuthenticationRegionInfo.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.specialized.cryptography;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class AuthenticationRegionInfo {
```

Comment:
consider naming it `AuthenticationRegion`
does this need to be public?

Reply:
Probably does not need to be public. I think this is naming that's used across languages, so I can consult with them. One thought it is not the actual region as it contains no data. It's just information about the region.